src/app/components/LiveLabel/index.styles.ts

```diff
@@ -22,17 +22,20 @@ const styles = {
       ...fontVariants.sansBold,
     }),
 
-  liveLabelCircle: ({ palette, spacings }: Theme) =>
+  pulseContainer: ({ spacings }: Theme) =>
     css({
-      color: palette.LIVE_DARK,
-      borderRadius: '50%',
       display: 'inline-block',
+      marginRight: '0.25em',
       width: `${spacings.DOUBLE}rem`,
       height: `${spacings.DOUBLE}rem`,
-      background: `radial-gradient(circle, transparent 53%, ${palette.LIVE_DARK} 53.5%, ${palette.LIVE_DARK} 54%)`,
-      marginInlineEnd: `${spacings.HALF}rem`,
-      position: 'relative',
-      transform: 'translate(0, 12%)',
+      verticalAlign: '-0.125em',
+    }),
+
+  pulsingCircle: () =>
+    css({
+      '@media (prefers-reduced-motion: no-preference)': {
+        animation: `${pulseAnimation} 1.25s cubic-bezier(0.455, 0.03, 0.515, 0.955) -0.4s 3`,
```

Comment:
Sorry to snoop on this - I wondered if this might also be an opportunity to improve the pulse behaviour? It sometimes feels like the pulse has already finished by the time you visit the page! I wonder if slowing down the pulse and starting a little bit later once the page loads might help?

I was doing a little bit of research on css animations, and there are 2 properties which affect the pulsation:
`animation-duration` (1.25s ☝️)  & `animation-delay` (-0.4s)  

If we were to tweak these values, the pulsation might last for a little bit longer e.g. duration 3s and delay 0.5s?

Should we also perhaps check with Catharine and UX about what the desired values here might be - we know that animations should not last for longer than 5s without the user being able to switch them off. I don't think we should increase the duration to 5s, but perhaps we could find a value which is acceptable for everyone? 

Just a thought 💭

Reply:
Absolutely, thank you, snooping always welcome 😆 I'll make a note of it to discuss between everyone at Monday's UX/a11y meeting and then hopefully settle on a final value before it's swarmed on again next week?

Reply:
Sounds good to me @karinathomasbbc great idea

Reply:
Although we agreed with UX not to change the pulse duration etc, can we change the offset to either 0 or 0.5s, so that we can test it during the a11y swarm?

```suggestion
        animation: `${pulseAnimation} 1.25s cubic-bezier(0.455, 0.03, 0.515, 0.955) 0.5s 3`,
```